src/main/java/ca/on/oicr/pineryreports/reports/impl/GazpachoProjectStatusReport.java

```diff
@@ -468,35 +507,32 @@ private static boolean hasDnaAnalyteName(SampleDto dto) {
         new ColumnDefinition(""),
         new ColumnDefinition(""),
         new ColumnDefinition(""),
+        new ColumnDefinition(""),
+        new ColumnDefinition(""),
+        new ColumnDefinition(""),
+        new ColumnDefinition(""),
+        new ColumnDefinition(""),
+        new ColumnDefinition(""),
         new ColumnDefinition(""));
   }
 
   @Override
   protected int getRowCount() {
-    switch (analyte) {
-    case ALL:
-      // print DNA and RNA on same sheet
-      return dnaRecentList.size()
-          + 3
-          + rnaRecentList.size()
+    int count = 1;
+    if (ALL.equals(analyte) || DNA.equals(analyte)) {
+      count += dnaRecentList.size()
           + 3
-          + dnaAllList.size()
+          + dnaStagnantList.size()
           + 3
-          + rnaAllList.size()
-          + 1;
-    case DNA:
-      return dnaRecentList.size()
+          + dnaAllList.size();
+    } else if (ALL.equals(analyte) || RNA.equals(analyte)) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this expression which always evaluates to "false" [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS2589)